### ISSUE TITLE
Text stored in HTML escaped version

### ISSUE THREAD

**Issue by edent** [edent/openbenches#118]
Text is stored - for example - as `Ralph &amp; Frances Maslen for many years enjoyed this park 2001`

A search for `ralph & fran` won't return anything.

**Comment by arizonagroovejet** [comment]
I have a cunning plan to fix this and #110. Possibly as good as one of Baldrick's.
Change this
https://github.com/edent/openbenches/blob/8ea9be3cc73a5b503f181226cfffb7cf2b0c8ed3/www/mysql.php#L638
to
```
        $searchLike =  "%".preg_replace('/\s+/', '%', $q)."%";
```

A bench with an & in the inscription and line breaks before and after it for good measure.
```
$ curl -k https://localhost/data.json/?bench=167;echo
var benches = {"type":"FeatureCollection","features":[{"id":167,"type":"Feature","geometry":{"type":"Point","coordinates":[4.108887,54.329342]},"properties":{"popupContent":"BUFFY\r\n&amp;\r\nSUMMERS\r\nHa see what I did there. &amp; because it sounds like Anne.\r\nIt'll become useful later, trust me.\r\n1981 - 2001\r\nBELOVED SISTER\r\nDEVOTED FRIEND\r\nSHE SAVED THE WORLD\r\nA LOT"}}]}
```
Searching for`buffy & summers` returns that bench.
![buffyampersandsummers](https://user-images.githubusercontent.com/29510692/46438835-d8032600-c756-11e8-8d62-96c16c2145f1.png)

**Comment by arizonagroovejet** [comment]
Yeah…that won't cause `<` `>` ``' `"` to be matched. :man_facepalming: Maybe it's fix for #110  Maybe…
